pkgs/by-name/ld/ld64/patches/0018-Add-dyldinfo-to-the-ld64-build.patch

```diff
@@ -0,0 +1,35 @@
+From bbe86ce2a2faff6525c487100ade3168175f8360 Mon Sep 17 00:00:00 2001
+From: Randy Eckenrode <randy@largeandhighquality.com>
+Date: Sun, 17 Nov 2024 10:02:34 -0500
+Subject: [PATCH 18/20] Add dyldinfo to the ld64 build
```

Comment:
Do we maybe want to make this a separate output since it isn’t present upstream any more and ~every build pulls in `ld64`?

Reply:
ld64 includes a few other binaries already. `dyldinfo` adds ~270 KiB.